v2/conn.go

```diff
@@ -49,7 +49,11 @@ type Conn struct {
 	RestartCounter uint8
 
 	// Sessions is a set of sessions exists on the Conn with automatically-assigned IDs.
-	Sessions []*Session
+	//Sessions []*Session
+
+	*IMSISessionMap
+
+	*TEIDSessionMap
```

Comment:
I don't think these fields should not be exported as it's no use for manipulating from outside the package(rather, might be dangerous), and the new `Session()` method satisfies the purpose of removed `Session` field.